### DISCUSSION TITLE
Can `tailsamplingprocessor` be configured to take the "sampled" flag (from `trace-flags`) into account?

### DISCUSSION THREAD

**Discussion by BatmanAoD** [open-telemetry/opentelemetry-collector-contrib#32495] [Q&A]
I have a service that is sending trace data to an OTEL collector, where it is sampled by the `tailsamplingprocessor` before being sent to Honeycomb. There is a client that does head-sampling, then sends its trace data directly to Honeycomb. I'd like to ensure that any calls that come from a client using head-sampling get sampled, i.e. that the `sampled` flag from the TraceContext's `trace-flags` is respected.

I can't tell if this is possible from the documentation of `tailsamplingprocessor`. @jpkrohling, in [this comment](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18462#issuecomment-1453419985), it sounds like you're saying that it's possible to use the `sampled` flag, but the README says:

> This processor must be placed in pipelines after any processors that rely on context, e.g. `k8sattributes`. It reassembles spans into new batches, causing them to lose their original context.

Does this mean that the `tailsamplingprocessor` doesn't receive the original trace context, including the `traceparent` header?

**Comment by jpkrohling** [comment]
I'll create an issue based on this and assign to me. I think it should be possible, but I'd need to test it out myself.
